content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -615,6 +616,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `LoggingBetaOptions`: Allow fine-tuing of experimental, beta-quality logging options.
 - `MatchLabelKeysInPodTopologySpread`: Enable the `matchLabelKeys` field for
   [Pod topology spread constraints](/docs/concepts/scheduling-eviction/topology-spread-constraints/).
+- `MatchLabelKeysInPodTAffinity`: Enable the `matchLabelKeys` and `mismatchLabelKeys` field for
+  [pod (anti)affinity](/docs/concepts/scheduling-eviction/assign-pod-node/).
```

Comment:
Same, please consider keeping these in alphabetical order. Also, there appears to be a typo in the Feature gate name (`MatchLabelKeysInPodTAffinity`) .